lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -652,6 +662,28 @@ WasmBinaryReader::FunctionBodyHeader()
     TRACE_WASM_DECODER(L"Function body header: i32 = %u, i64 = %u, f32 = %u, f64 = %u", i32Count, i64Count, f32Count, f64Count);
 }
 
+void
+WasmBinaryReader::ImportEntry()
+{
+    UINT len = 0;
+    UINT sigId = LEB128(len);
+
+    m_funcInfo = Anew(&m_alloc, WasmFunctionInfo, &m_alloc);
+    m_funcInfo->SetModuleName(ReadString());
+    m_funcInfo->SetName(ReadString());
+
+    m_currentNode.func.info = m_funcInfo;
```

Comment:
this is not used... we should create another table of WasmFunctionInfo for imports on ModuleInfo.